console/console-init/ui/src/modules/iot-device-detail/components/DeviceInfo/DeviceInfo.tsx

```diff
@@ -45,12 +47,16 @@ export interface IDeviceInfoProps
     IErrorStateAlertProps {
   id: string;
   deviceList?: any;
+  deviceGroup?: any;
+  memberGroup?: any;
   metadataList?: any;
 }
 
 export const DeviceInfo: React.FC<IDeviceInfoProps> = ({
   id,
   deviceList,
+  deviceGroup,
```

Comment:
deviceGroup->viaGroups
memberGroup-> memberOf

Reply:
I have changed  device group to gatewayGroups and memberGroup-> memberOf